packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -13,44 +15,53 @@ import {
 } from "../utilities/color";
 import { applyFloatingCornerRadius } from "../utilities/border";
 import { designUnit } from "../utilities/design-system";
-
-export const selectDisplayButtonOverrides: ComponentStyles<
-    Partial<ButtonClassNameContract>,
-    DesignSystem
-> = {
-    button: {
-        width: "100%",
-        padding: "0 10px",
-    },
-    button_contentRegion: {
-        width: "100%",
-        display: "inline-flex",
-    },
-};
+import { inputFieldStyles } from "../patterns/input-field";
+import { applyDisabledState } from "../utilities/disabled";
+import { focusOutlineWidth } from "../utilities/design-system";
 
 const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<SelectClassNameContract, DesignSystem> => {
-    const designSystem: DesignSystem = withDesignSystemDefaults(config);
-
     return {
         select: {
             minWidth: "276px",
             maxWidth: "374px",
         },
 
+        select_button: {
+            height: height(),
+            width: "100%",
+            ...inputFieldStyles(),
+        },
+
+        select_button_contentRegion: {
+            display: "grid",
+            gridTemplateColumns: "1fr auto",
+            alignItems: "center",
+            justifyItems: "start",
+        },
+
+        select_button_displayText: {
+            whiteSpace: "nowrap",
+            overflow: "hidden",
```

Comment:
I think there's a helper for this. We should probably use ellipsis.

Reply:
done